src/main/java/com/hollingsworth/arsnouveau/common/ritual/RitualAwakening.java

```diff
@@ -19,26 +20,37 @@
 import net.minecraft.world.entity.EntityType;
 import net.minecraft.world.entity.LivingEntity;
 import net.minecraft.world.entity.item.ItemEntity;
+import net.minecraft.world.entity.player.Player;
 import net.minecraft.world.item.ItemStack;
 import net.minecraft.world.item.WritableBookItem;
 import net.minecraft.world.level.Level;
 import net.minecraft.world.level.block.Blocks;
 import net.minecraft.world.phys.Vec3;
+import net.neoforged.neoforge.common.util.FakePlayer;
+import org.jetbrains.annotations.Nullable;
 
 import java.util.Set;
+import java.util.UUID;
 
 public class RitualAwakening extends AbstractRitual {
 
     EntityType<? extends LivingEntity> entity = null;
     BlockPos foundPos;
+    UUID uuid;
 
-    public void destroyTree(Level world, Set<BlockPos> set) {
+    @Override
+    public void onStart(@Nullable Player player) {
+        if (player != null) uuid = player.getUUID();
+    }
+
+    public void destroyTree(ServerLevel world, Set<BlockPos> set) {
+        FakePlayer fakePlayer = ANFakePlayer.getPlayer(world, uuid);
         for (BlockPos p : set) {
-            BlockUtil.destroyBlockSafelyWithoutSound(world, p, false);
+            BlockUtil.destroyBlockSafelyWithoutSound(world, p, false, fakePlayer);
```

Comment:
This is technically causing a dupe here if we aren't going to check that a block was successfully destroyed before we spawn the entity.